system/trace/trace_dump.c

```diff
@@ -128,11 +128,11 @@ static void note_ioctl(int cmd, unsigned long arg)
 {
   int notefd;
 
-  notefd = open("/dev/note", O_RDONLY);
+  notefd = open("/dev/note/ram", O_RDONLY);
   if (notefd < 0)
     {
       fprintf(stderr,
-             "trace: cannot open /dev/note\n");
+             "trace: cannot open /dev/note/ram\n");
```

Comment:
Optional:
can be one line instead of two

Reply:
done